config.json

```diff
@@ -103,220 +86,246 @@
         ]
       },
       {
-        "slug": "difference-of-squares",
-        "name": "Difference of Squares",
-        "uuid": "9802cbc0-eb70-4840-b8ee-a228f1fc29ea",
+        "slug": "resistor-color",
+        "name": "Resistor Color",
+        "uuid": "75653d83-1a64-472c-ac43-49e87833c7f7",
         "practices": [],
         "prerequisites": [],
         "difficulty": 1,
         "topics": [
-          "control_flow_loops",
+          "arrays",
+          "enums",
+          "pointers"
+        ]
+      },
+      {
+        "slug": "collatz-conjecture",
+        "name": "Collatz Conjecture",
+        "uuid": "5a47f8a5-36a1-4479-bb77-ef5fbab4bae0",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2,
+        "topics": [
+          "functions",
           "math"
         ]
       },
       {
-        "slug": "square-root",
-        "name": "Square Root",
-        "uuid": "0400f52d-48ae-4300-a72e-32e8a08514d0",
+        "slug": "queen-attack",
+        "name": "Queen Attack",
+        "uuid": "b0a152e9-5a45-41f9-bda0-427111d9a56c",
         "practices": [],
         "prerequisites": [],
-        "difficulty": 4,
+        "difficulty": 2,
         "topics": [
-          "bitwise_operations",
-          "control_flow_loops",
+          "control_flow_if_statements",
+          "enums",
+          "functions",
+          "structs"
+        ]
+      },
+      {
+        "slug": "darts",
+        "name": "Darts",
+        "uuid": "a579e471-86f3-4f29-82ca-e984a2cc8891",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2,
+        "topics": [
+          "control_flow_if_else_statements",
+          "logic",
           "math"
         ]
       },
       {
-        "slug": "grade-school",
-        "name": "Grade School",
-        "uuid": "68b1de28-7c3d-4e8c-ab51-7abe6779e187",
+        "slug": "resistor-color-duo",
```

Comment:
I would have thought that this would come directlay after `resistor-color`, but it is a while since I've looked at the exercise.
Can you share why it isn't'?

Reply:
No reason. I'll move it around.

Reply:
Actually, because I rated them 1, 2, and 3 I can't get them to be right next to each other without having the difficulties out of order. I guess a benefit of them not being in a row is that people won't tired of resistors.

Reply:
Seems like a good enough reason, maybe leave them as they are then and we can see if we get any feedback.